### PR TITLE
MAID-3113: docs/license: update LICENSE details

### PR DESCRIPTION
**Pull request by pravarseth**

### CODE REVIEW

File: LICENSE-BSD [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Redistribution and use in source and binary forms, with or without
```
Comment (Fraser999) [inline LICENSE-BSD:3]: Shouldn't this have copyright info like `Copyright 2018 MaidSafe.net limited.` at the start? (See https://opensource.org/licenses/BSD-3-Clause)

Same for the other copy of this in the -sys lib.

**Review by Fraser999 — DISMISSED** [review]
Should we consider adding our standard copyright block to the top of all source files too?

File: LICENSE-BSD (hunk 1 above)
Comment (nadiaburborough) [reply LICENSE-BSD]: @NickLambert thoughts on the above please?

Comment (nadiaburborough) [reply LICENSE-BSD:3]: Yes it should. If you approve this, I'll raise a PR to ammend it @Fraser999

**Review by S-Coyle — APPROVED** [review]
👍

**Pull request merged**